src/global.d.ts

```diff
@@ -2,3 +2,5 @@ declare module '*.scss' {
   const content: {[className: string]: string};
   export = content;
 }
+
+declare module 'humanize-duration';
```

Comment:
import doesn't work without adding this ? weird

Reply:
no, need to add `@ts-ignore` if not adding this.